.github/workflows/README.md

```diff
@@ -1,56 +1,22 @@
-# CBMC packages
```

Comment:
Should this information, rather than being removed, be moved to some other place? Perhaps even be merged into the yaml file that takes care of building installation packages?

Reply:
I do not think so. Most of that information is no longer relevant, and the relevant pieces of information (installation instructions) are present on both our release pages and the top-level `README.md` file.